main.js

```diff
@@ -0,0 +1,251 @@
+    tags: {
+      tag1: 'HTML',
+      tag2: 'CSS',
+      tag3: 'JS',
+      tag4: 'React',
+      tag5: 'Ruby',
+    },
+    linkLive: '#',
+    linkSource: '#',
+  },
+  {
+    title: 'Project 4: Keeping track of hundreds of components',
+    image:
+      'https://raw.githubusercontent.com/RaoAkif/MicroversePortfolio/main/images/portfolio_modal/Portfolio_Projects%20-%20Mobile%20Modal_4.png',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release.",
+    tags: {
+      tag1: 'HTML',
+      tag2: 'CSS',
+      tag3: 'JS',
+      tag4: 'React',
+      tag5: 'Ruby',
+    },
+    linkLive: '#',
+    linkSource: '#',
+  },
+  {
+    title: 'Project 5: Keeping track of hundreds of components',
+    image:
+      'https://raw.githubusercontent.com/RaoAkif/MicroversePortfolio/main/images/portfolio_modal/Portfolio_Projects%20-%20Mobile%20Modal_5.png',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release.",
+    tags: {
+      tag1: 'HTML',
+      tag2: 'CSS',
+      tag3: 'JS',
+      tag4: 'React',
+      tag5: 'Ruby',
+    },
+    linkLive: '#',
+    linkSource: '#',
+  },
+  {
+    title: 'Project 6: Keeping track of hundreds of components',
+    image:
+      'https://raw.githubusercontent.com/RaoAkif/MicroversePortfolio/main/images/portfolio_modal/Portfolio_Projects%20-%20Mobile%20Modal_6.png',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release.",
+    tags: {
+      tag1: 'HTML',
+      tag2: 'CSS',
+      tag3: 'JS',
+      tag4: 'React',
+      tag5: 'Ruby',
+    },
+    linkLive: '#',
+    linkSource: '#',
+  },
+];
```

Comment:
[OPTIONAL] To keep the functionality/logic and data separate, consider putting the data in a separate file.